integration-cli/docker_cli_build_test.go

```diff
@@ -1986,8 +1986,7 @@ func TestBuildCancelationKillsSleep(t *testing.T) {
 	eventStart := make(chan struct{})
 	eventDie := make(chan struct{})
 
-	// Start one second ago, to avoid rounding problems
-	startEpoch := time.Now().Add(-1 * time.Second)
+	startEpoch := daemonTime(t).Unix()
```

Comment:
Because https://github.com/docker/docker/pull/12090/files#diff-9e07f7c1f4d16eb428d628f85ee1a6f1R417
there isn't need for -1 there and I think here isn't needed too. This just need a refactor to prevent races.